packages/@webex/plugin-meetings/src/reconnection-manager/index.ts

```diff
@@ -443,17 +443,19 @@ export default class ReconnectionManager {
       }
     }
 
-    try {
-      LoggerProxy.logger.info(
-        'ReconnectionManager:index#executeReconnection --> Updating meeting data from server.'
-      );
-      await this.webex.meetings.syncMeetings();
-    } catch (syncError) {
-      LoggerProxy.logger.info(
-        'ReconnectionManager:index#executeReconnection --> Unable to sync meetings, reconnecting.',
-        syncError
-      );
-      throw new NeedsRetryError(syncError);
+    if (!this.webex.credentials.isUnverifiedGuest) {
```

Comment:
suggestion: I know this function is already kind of messy in general and does like 5 things in sequence, but as it's growing here and now there is conditional logic, would it not make sense to create a separate function, something like, `reconnectionSync(this.webex.credentials)`

where reconnectionSync looks like

`const reconnectionSync = (options: {isUnverifiedGuest: boolean, overrideSync, etc:}`

and it handles all of this type of stuff, that way this function can slowly become sort of a dispatcher and then the new function could be tested in isolation.

Obviously it's fine as is because we have access to webex.credentials, but just thinking about moving towards cleaning this function up.

Reply:
At some point yes it would be nice to have this function call other functions rather than doing all this extra work, but given the lack of existing unit tests I feel more comfortable introducing minimal changes here rather than splitting up as part of this bug fix.